recipes/cppadcodegen/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "cppadcodegen" %}
+{% set version = "2.4.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/joaoleal/CppADCodeGen/archive/v{{ version }}.tar.gz
+  sha256: 3d3fee920f67b70566fa09ba62203323d4a8100d1ab210c6e7b34f7cb23349fb
+
+build:
+  number: 0
+  # No ABI policy documented, let's be conservative 
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [linux]
+  host:
+    - cppad
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/cppad/cg  # [unix]
+    - test -f ${PREFIX}/share/pkgconfig/cppadcg.pc  # [unix]
+    - if not exist %PREFIX%\\Library\\include\\cppad\cg exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\share\\pkgconfig\\cppadcg.pc exit 1  # [win]
+
+about:
+  home: https://github.com/joaoleal/CppADCodeGen
+  license: GPL-3.0-only AND EPL-2.0 
```

From https://github.com/joaoleal/CppADCodeGen#license I guess that you can choose either license, so perhaps the correct license is `OR` .
```suggestion
  license: GPL-3.0-only OR EPL-2.0 
```

---

Thanks